src/QirRuntime/test/Standalone/Shared/CLI11.hpp

```diff
@@ -0,0 +1,8258 @@
+#pragma once
```

Comment:
Why this file is needed?

Reply:
It is a [command line parser utility](https://github.com/CLIUtils/CLI11) used for getting the arguments that need to be passed to the QIR entry-point (work in progress).

Reply:
When incorporating external code we probably need to:
- confirm the licensing terms and acknowledge it in the licensing manifest of our repo
- do a security review
- provide some way of versioning for maintenance and being able to pick up the updates
- consider: if this is a sample for end users, is it OK to force _them_ to take the dependency?
- - ???

The bottom line is, we shouldn't take external dependencies lightly. Do we have a precedent that involves CLI? Who can review this from the point of view of component governance?

Reply:
@cesarzc, FYI: We have a `qsharp-runtime\src\QirRuntime\externals` folder for external components.

Reply:
Followed up on this offline, added it to the manifest, moved it to the `externals` folder, and we should be good to go now. We have to add it to NOTICE.txt once we start distributing it as part of the QDK.